README.md

```diff
@@ -104,6 +106,8 @@ vmap     <C-F>F <Plug>CtrlSFVwordExec
 nmap     <C-F>n <Plug>CtrlSFCwordPath
 nmap     <C-F>p <Plug>CtrlSFPwordPath
 nnoremap <C-F>o :CtrlSFOpen<CR>
+nnoremap <silent><C-F>t :CtrlSFToggle<cr>
+imap <C-F>t <Esc>:CtrlSFToggle<cr>
```

Comment:
Please align these two lines, and I think `<silent>` is unnecessary. (It's not something bad but better to delete it for... um, alignment)